packages/smart-wallet/src/smartWallet.js

```diff
@@ -596,15 +594,12 @@ export const prepareSmartWallet = (baggage, shared) => {
           }
 
           // Accept the issuer; rely on it in future offers.
-          const [displayInfo, purse] = await Promise.all([
-            E(issuer).getDisplayInfo(),
-            E(issuer).makeEmptyPurse(),
-          ]);
+          const [purse] = await Promise.all([E(issuer).makeEmptyPurse()]);
```

Comment:
```suggestion
          const [purse] = await E(issuer).makeEmptyPurse();
```

I doubt there is ever any point to a singleton call to `Promise.all`

Reply:
yeah, this was just a draft
```suggestion
          const purse = await E(issuer).makeEmptyPurse();
```

Reply:
I like the overall approach. I love that you moved the ERTP types, with comments, from types.js to types.d.ts. typedoc FTW!

Beyond that, I like all the changes outside ERTP, including the FIXME question. I'm curious about the answer. Because I'm not sure that deprecating `brand.getDisplayInfo()` is the right thing to do. But I find it plausible.

I recommend proceeding forward. Thanks.

Reply:
> I doubt there is ever any point to a singleton call to Promise.all

I think of it as akin to the dangling comma on single-argument method calls, i.e. to reduce churn in formatting in places were there were multiple operands, and might be again.  Do people believe that automated reformatting makes that concern moot?

Reply:
moot enough, yes. The only remaining gotcha is git's text-line based diff and merge algorithms. But IMO we should not distort our programs too much to accommodate these algos being at the wrong abstraction level. Focus on the human reader/reviewer first.